src/entity/user/UserMarketingCta.ts

```diff
@@ -0,0 +1,30 @@
+import { Column, Entity, ManyToOne, PrimaryColumn } from 'typeorm';
+import { User } from './User';
+import { MarketingCta } from '../MarketingCta';
+
+@Entity()
+export class UserMarketingCta {
+  @PrimaryColumn({ type: 'text' })
+  marketingCtaId: string;
+
+  @PrimaryColumn({ length: 36 })
```

Comment:
We can just use `text`, it both uses the same [underlying data structure](https://www.depesz.com/index.php/2010/03/02/charx-vs-varcharx-vs-varchar-vs-text/).